common/api-review/firestore-exp.api.md

```diff
@@ -521,6 +521,9 @@ export function updateDoc(reference: DocumentReference<unknown>, data: UpdateDat
 // @public
 export function updateDoc(reference: DocumentReference<unknown>, field: string | FieldPath, value: unknown, ...moreFieldsAndValues: unknown[]): Promise<void>;
 
+// @public
+export function useFirestoreEmulator(firestore: FirebaseFirestore_2, host: string, port: number): void;

```

Comment:
Is the `_2` intentional?

Reply:
Yeah, it's because we share the code from firestore-lite and firestore-exp. This should go away once https://github.com/firebase/firebase-js-sdk/pull/4140 is merged.